proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,566 @@
+3. Sie entscheidet über Einsprüche gegen den Ausschluss von Mitgliedern.
+4. Sie nimmt den Jahresbericht des Vorstandes und den Bericht der
+   Rechnungsprüfer entgegen und erteilt dem Vorstand Entlastung.
+5. Sie beschließt über Satzungsänderungen und die Auflösung des Vereins.
+6. Ihr sind insbesondere die Jahresrechnung und der Jahresbericht zur
+   Beschlussfassung über die Genehmigung und die Entlastung des Vorstandes
+   schriftlich vorzulegen. Sie beruft einen Rechnungsprüfer, die weder dem
+   Vorstand noch einem vom Vorstand berufenen Gremium angehören und auch nicht
+   beim Verein angestellt ist, um die Buchführung einschließlich
+   Jahresabschluss zu prüfen und über das Ergebnis vor der
+   Mitgliederversammlung zu berichten. Dr Rechnungsprüfer hat Zugang zu allen
+   Buchungs- und Rechnungsunterlagen des Vereins.
+7. Sie beschließt über Käufe, Verkäufe, Investitionen und die Aufnahme von
+   Krediten.
+
+## 8. Der Vorstand
+
+1. Der Vorstand besteht aus dem Vorsitzenden, dem zweiten Vorsitzenden und dem
+   Schatzmeister, die für eine Amtszeit von 3 Jahren gewählt werden.
+2. Die Aufgaben des Vorstands und der Ablauf der Sitzungen wird durch eine
+   Geschäftsordnung des Vorstands geregelt. Der Vorsitzende ist dabei
+   für die Verfolgung des Vereinszwecks, die Einhaltung der Satzung und die
+   Delegation von Aufgaben verantwortlich. Der zweite Vorsitzende vertritt den
+   Vorsitzenden bei dessen Abwesenheit und überprüft die Beschlüsse des
+   Vorstands und der Mitgliederversammlungen. Der Schatzmeister verwaltet die
+   Finanzen des Vereins, führt die Kassenbücher und die Mitgliederliste.
+3. Der Vorstand beschließt über alle Vereinsangelegenheiten, soweit sie nicht
+   eines Beschlusses der Mitgliederversammlung bedürfen. Er führt die
+   Beschlüsse der Mitgliederversammlung aus.
+4. Jedes Vorstandsmitglied hat das Recht, den Verein zu vertreten.
+5. Im Falle des Ausscheidens eines Vorstandsmitglieds während der Amtszeit
+   bestimmt der Vorstand ein kommissarisches Vorstandsmitglied. Das
+   kommissarische Vorstandsmitglied bleibt bis zur nächsten
+   Mitgliederversammlung im Amt. Über eine endgültige Nachfolge im
+   Vorstand entscheidet die Mitgliederversammlung.
+6. Im Falle des Ausschlusses eines Vorstandsmitglieds gemäß §4 (5) wird dieses
+   durch ein kommissarisches Vorstandsmitglied ersetzt.
+7. Der Vorstand kann zur Führung der laufenden Geschäfte einen Geschäftsführer
+   bestellen.
+8. Der Geschäftsführer nimmt an den Mitgliederversammlungen und
+   Vorstandssitzungen teil.
+9. Der Vorstand kann die Satzung aus formalen Gründen ändern, wenn von den
+   Behörden verlangt wird. Über die Änderung wird in der darauffolgenden
+   Mitgliederversammlung abgestimmt.
+
+## 9. Geschäftsordnung
+
+Die Geschäftsordnung regelt über den Rahmen der Satzung hinausgehende Details
+der Vereinsarbeit. Der Vorstand setzt die Geschäftsordnung nach der Zustimmung
+der Kernmitglieder in einer Internetabstimmung in Kraft.
```

Comment:
right, then I think we should reformulate this sentence in terms of that. Otherwise this looks a little weird and suspicious and could become a loophole.

Reply:
With "in Kraft setzen" is the initial version references. This will happen short after the founding General Assembly and therefore an internet vote is IMHO appropriated. We have not adopted sentences KDE e.V 8.3 about 8.3 Board is able to change the By law if it is done unanimously by vote. It seems to be a common practice to define the mode of change in the By law itself. In Templates you find also versions where a majority of board members is sufficient for a change.  Is think changes of the by law should not required a General assembly to keep the ability to act. We must to make sure that all loophole are removed in this document which is hard to change after we are founded. 

Does that make sense? Do we want to amend this paragraph?

Reply:
I afraid I can't follow you. My initial proposal was just to replace "Kernmitglieder" with "Mitgliederversammlung" because that sounded nicer and afai understand they're equivalent in terms of decision power, but on second thought it doesn't make much sense.

Reply:
First of all the sentence alone should be understandable. This is probably not yet the case, so please propose a change if you have one in mind. 

In general we have three decision making levels 
* The Board 
* The Internet vote of core members 
* The Gneral Assembly. 

The General Assembly meets once a year, and this  meeting requires quite a bit of formalism. 
The internet vote is a quick thing with the same obligation and the same group of core members. 
The Board are only three people 

In the case of bringing the first by law in force a discussion like in this PR + a vote is probably sufficient. 
= "Kernmitglieder in einer Internetabstimmung"

What we not have discussed is the topics the by-law covers and the mode to iterate it. 
It is the common practice that the Board is able to adjust it on their needs because it may effects their day to day works. We may decide in the by law itself about the core member involvement. If you are afraid of a loophole we may define it here with the risk of hard later changes. 
Fundamental risk for Mixxx or the Association should be covered in this document before the founding, because this is the one that can be only changed with core member involvement and after authorities review.

Reply:
No worries, everything makes sense now.